src/client/Cargo.toml

```diff
@@ -29,7 +29,7 @@ tokio = { version = "0.2.21", features = ["macros"] }
 # Fluvio dependencies
 fluvio-future = { version = "0.1.0", features = ["tls"] }
 fluvio-types = { version = "0.1.0", path = "../types" }
-fluvio-sc-schema = { version = "0.1.0", path = "../sc-schema", default-features = false }
+fluvio-sc-schema = { version = "0.1.0", path = "../sc-schema", features = ["use_serde"] }
```

Comment:
@sehz  This also appears to be how the CLI is using the features, so not sure we want to add a feature gate around this. I think most clients will need this feature.

Reply:
I think serde is used by admin api.  Not need for consumer or producer

Reply:
If we _do_ want to add a feature gate around the admin API, then I think we should make it specifically for that. We can make a feature called `admin` or `fluvio-admin` which depends on a `serde` as an optional feature if we want.

Reply:
commit b7bef94 adds `admin` feature.